software/filephile.livecodescript

```diff
@@ -0,0 +1,150 @@
+﻿script "filephile"
+on devLog pPhase
+   --   -- ! commented out 15DEC2015 cuz logging slows things down a lot
```

Comment:
@matthewmaier I often use a boolean constant like kDebug to govern whether I want to do things like logging. The other thing here is the reason why this is slowing things down is the way you are writing to the file. Each log you are reading the entire file in, counting the number of lines, appending a new line and then writing it all out. I would change this in the following ways:

* include the date in the log file name so you can just look at todays logs
* the first log in the session I would `open file logFilePath for "utf8" text append`
* write a line for the log each time you want to log (this will append at the end without reading the whole file into memory)
* close the file when quitting your app